src/drivers/imu/bosch/bmi088_i2c/BMI088_Accelerometer.cpp

```diff
@@ -908,11 +913,29 @@ bool BMI088_Accelerometer::NormalRead(const hrt_abstime &timestamp_sample)
 	const int16_t accel_y = combine(RATE_Y_MSB, RATE_Y_LSB);
 	const int16_t accel_z = combine(RATE_Z_MSB, RATE_Z_LSB);
 
+	if (accel_x == INT16_MIN) {
+		PX4_WARN("accel_x == INT16_MIN");
```

Comment:
Generally don't print messages in drivers, they can run at high rate, spam the console, and even hurt realtime performance.

If you want to track anything more like this I'd suggest adding perf counters.

Reply:
Agreed. This was a temporary measure to see if CrazyFlie test (if ever happening) would produce any warnings. BTW, this happens at low rate randomly and doesn't overwhelm the console.